### PR TITLE
New Player Panel: Current Character

### PR DESCRIPTION
**Pull request by XSlayer300**
## About The Pull Request
Adds a "Current character: [NAME]" below the observe button, simple as that.

https://cdn.discordapp.com/attachments/500801682058379265/665527011292348428/unknown.png

## Why It's Good For The Game
To inform human players to know what you are playing as before roundstart or latejoin.

## Changelog
:cl:
tweak: Edited the New Player Panel for Current Character
/:cl:

### CODE REVIEW

File: code/datums/gamemodes/distress.dm [hunk 1]
```diff
@@ -466,6 +466,7 @@
 	output +="<hr>"
 	output += "<p><a href='byond://?src=[REF(NP)];lobby_choice=show_preferences'>Setup Character</A> | <a href='byond://?src=[REF(NP)];lobby_choice=lore'>Background</A><br><br><a href='byond://?src=[REF(NP)];lobby_choice=observe'>Observe</A></p>"
 	output +="<hr>"
+	output += "<center><p>Current character: <b>[NP.client ? NP.client.prefs.real_name : "Unknown User"]</b></p>"
```
Comment (psykzz) [inline code/datums/gamemodes/distress.dm]:
```suggestion
	output += "<center><p>Current character: <b>[NP.client ? NP.client.prefs.real_name : "Unknown User"]</b></p></center>"
```

File: code/datums/gamemodes/crash.dm [hunk 2]
```diff
@@ -268,6 +268,7 @@
 	output +="<hr>"
 	output += "<p><a href='byond://?src=[REF(NP)];lobby_choice=show_preferences'>Setup Character</A> | <a href='byond://?src=[REF(NP)];lobby_choice=lore'>Background</A><br><br><a href='byond://?src=[REF(NP)];lobby_choice=observe'>Observe</A></p>"
 	output +="<hr>"
+	output += "<center><p>Current character: <b>[NP.client ? NP.client.prefs.real_name : "Unknown User"]</b></p>"
```
Comment (psykzz) [inline code/datums/gamemodes/crash.dm]:
```suggestion
	output += "<center><p>Current character: <b>[NP.client ? NP.client.prefs.real_name : "Unknown User"]</b></p></center>"
```

**Comment by Rohesie** [comment]
The panel doesn't update when the character is changed.

**Comment by XSlayer300** [comment]
I directly copied it from TG, so it has the same bug too. Sadly.

**Pull request merged**